core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractProject.java

```diff
@@ -469,7 +473,7 @@ public int hashCode() {
         .append(alias)
         .append(description)
         .append(progress)
-        .append(referenceGenomeId)
+        .append(referenceGenome.getId())
```

Comment:
Potential NPE. Also won't be useful when the ID is not set (such as during migration). Could just append the ReferenceGenome object here, and then base ReferenceGenome's hashCode on the alias (which is a unique key also)